frontend/lib/src/components/widgets/FileUploader/FileDropzone.tsx

```diff
@@ -41,39 +41,41 @@ const FileDropzone = ({
   disabled,
   label,
 }: Props): React.ReactElement => (
-  <Dropzone
-    onDrop={onDrop}
-    multiple={multiple}
-    accept={acceptedExtensions.length ? acceptedExtensions : undefined}
-    maxSize={maxSizeBytes}
-    disabled={disabled}
-    // react-dropzone v12+ uses the File System Access API by default,
-    // causing the bug described in https://github.com/streamlit/streamlit/issues/6176.
-    useFsAccessApi={false}
-  >
-    {({ getRootProps, getInputProps }) => (
-      <StyledFileDropzoneSection
-        {...getRootProps()}
-        data-testid="stFileUploadDropzone"
-        isDisabled={disabled}
-        aria-label={label}
-      >
-        <input {...getInputProps()} />
-        <FileDropzoneInstructions
-          multiple={multiple}
-          acceptedExtensions={acceptedExtensions}
-          maxSizeBytes={maxSizeBytes}
-        />
-        <BaseButton
-          kind={BaseButtonKind.SECONDARY}
-          disabled={disabled}
-          size={BaseButtonSize.SMALL}
+  <div data-testid="stDropzone">
```

Comment:
Do we need a new parent div here? Can't `data-testid="stFileDropzone"` just be added to the `<Dropzone>` component below?

Reply:
I attempted to add a `data-testid` and that didn't work. In addition, i don't think there are props within the react-dropzone in order to override the data-testid: https://react-dropzone.js.org/

Reply:
Examined and looks like it doesn't accept this as a prop bc `Dropzone` doesn't translate to an element rendered in the DOM. I'd prefer to remove this outer div/data-testid and just test that `data-testid="stFileUploadDropzone"` is in the document, since it is essentially the outermost/parent component for this file:
![Screenshot 2023-09-15 at 12 15 24 PM](https://github.com/streamlit/streamlit/assets/63436329/7e80d8a8-4326-4c07-8955-1a9ce7863fe2)